### DISCUSSION TITLE
OpenDeck Configurator, define which CC# is sent by e.g. analog inputs?

### DISCUSSION THREAD

**Discussion by stllvdv** [shanteacontrols/OpenDeck#213] [Q&A] [answered]
Hey @paradajz et al,

Currently exploring [OpenDeck Configurator](https://config.shanteacontrols.com/) and got my first knob sending! So happy right now! Now wondering… MIDI-learn works like a charm already. Is also possible to pre-define which CC# are sent by e.g. the analog inputs?

Warm regards!

**Comment by paradajz** [comment]
Not sure what do you mean with pre-define?

**Comment by stllvdv** [comment, marked as answer]
My bad, that's what MIDI-ID defines — obviously! Missed that…
